lms/templates/ccx/student_admin.html

```diff
@@ -2,10 +2,19 @@
 
 <section>
   <h2>${_('Student Grades')}</h2>
-  <p>
-    <a href="${gradebook_url}">${_('View gradebook')}</a>
-  </p>
-  <p>
+  <p class="ccx-grade-book-download">
+    <i class="fa fa-download" aria-hidden="true"></i>
     <a href="${grades_csv_url}">${_('Download student grades')}</a>
   </p>
+  <br/>
+  <div>
+    <iframe src='${gradebook_url}' id="ccx_grade_book_iframe"
+            name="ccx_grade_book_iframe"
+            frameBorder="0"
+            scrolling="no"
+            height="360px"
+            class="ccx-grade-book">
+      <p>Iframe not supported</p>
+    </iframe>
```

Comment:
OK, thanks for the clarification. I doubt iframe is the right way to go, but I'd like to understand how this is used and what you're really going for here before saying for sure. It may be easier to just arrange a 10 minute chat sometime. Why don't you shoot me an email with a time that's good for you - I'm pretty open Friday and next week M-W.